lib/rules/property-allowed-list/README.md

```diff
@@ -4,39 +4,40 @@ Specify a list of allowed properties.
 
 <!-- prettier-ignore -->
 ```css
-a { display: block; }
+a { color: red; }
 /** ↑
  * This property */
 ```
 
-This rule ignores variables (`$sass`, `@less`, `--custom-property`).
+This rule ignores preprocessor variables (`$sass`, `@less`).
```

Comment:
```suggestion
This rule ignores preprocessor variables (e.g. `$sass`, `@less`).
```

″ for property-disallowed-list/README.md

Reply:
I updated the few occurrences. Thank you.